djangocms_versioning/datastructures.py

```diff
@@ -78,17 +78,20 @@ def version_model_proxy(self):
     def grouper_model(self):
         return self.grouper_field.remote_field.model
 
-    def distinct_groupers(self):
+    def distinct_groupers(self, **kwargs):
         """Returns a queryset of `self.content` objects with unique
         grouper objects.
 
         Useful for listing, e.g. all Polls.
+
+        Args:
+            kwargs: Optionally it takes dict object (**kwargs)
```

Comment:
Please improve docstring here. Something like "optional filtering parameters for inner queryset". Also the format is wrong, we aren't using google-style param formatting